python/ray/includes/gcs_client.pxi

```diff
@@ -595,6 +745,77 @@ cdef raise_or_return(tup):
 # Must not raise exceptions, or it crashes the process.
 #############################################################
 
+# IDEA(ryw): use Cython fused types to reduce boilerplate.
+cdef convert_optional_raw_get_all_actor_info_reply(
+        CRayStatus status, optional[CGetAllActorInfoReply]&& c_data) with gil:
+    # -> gcs_service_pb2.GetAllActorInfoReply
```

Comment:
we can't just use this as an output type?

Reply:
Cython limitations... can't make it right to return typed values